implementations/micrometer-registry-statsd/build.gradle

```
@@ -10,6 +10,7 @@ dependencies {
         // We do not use HTTP modules; exclude to avoid false positive CVE reports
         exclude module: 'netty-codec-http'
     }
+    implementation(group: 'io.netty', name: 'netty-transport-native-epoll', classifier: 'linux-aarch_64')
```

In order to make this explicit, could you please also add the x86 variant?
(we did the same with dns-macos)
```suggestion
    implementation(group: 'io.netty', name: 'netty-transport-native-epoll', classifier: 'linux-aarch_64')
    implementation(group: 'io.netty', name: 'netty-transport-native-epoll', classifier: 'linux-x86_64')
```